openupgradelib/openupgrade.py

```diff
@@ -1031,13 +1032,16 @@ def migrate(no_version=False, use_env=None, uid=None, context=None):
     logging purposes.
     """
     def wrap(func):
-        def wrapped_function(cr, version):
+        @wraps(func)
+        def wrapped_function(cr, version, no_version=no_version, use_env=use_env,
+                             uid=uid, context=context):
```

Comment:
Ahh. Some enlightenment comes to my mind. Thanks for the explanation! :+1: 
So you would rather say:
Not doing reassignment in the function signature, but avoid naming clashes?